cms/djangoapps/contentstore/config/forms.py

```diff
@@ -0,0 +1,37 @@
+"""
+Defines a form for providing validation.
```

Comment:
All this form does is make it impossible to enter an invalid/nonexistent course_id for the course-specific flag.